src/sentry/interfaces/stacktrace.py

```diff
@@ -140,8 +140,11 @@ def is_newest_frame_first(event):
     return newest_first
 
 
+scheme_prefix_re = re.compile('^([a-z]+[a-z+-\.]*)://', re.IGNORECASE)
```

Comment:
Might be nice to see a comment here calling out that this is specifically tuned to RFC 3986 § 3.1

```
# Matches: https://tools.ietf.org/html/rfc3986#section-3.1
```

Reply:
You just wanted to use the section glyph in a comment, be honest

Reply:
<kbd>◇6</kbd>